src/services/control-plane/adapters/drivers/managing-authentication-proxy-adapter.ts

```diff
@@ -0,0 +1,13 @@
+import { ControlPlane } from '../../app/control-plane';
+import { AuthDetails,Permissions } from '../../app/schemas';
+import { ForManagerAuthentication } from '../../ports/drivers';
+
+export class ManagerAuthenticationProxyAdapter implements ForManagerAuthentication{
+     constructor(private readonly controlPlane: ControlPlane){}
```

Comment:
Agregar espacios entre unidades lógicas
```suggestion
     constructor(private readonly controlPlane: ControlPlane){}
     
```